internal/telemetry/traces/chunking_test.go

```
@@ -51,19 +55,21 @@ func traceAPIResponse(n int) string {
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		requests = append(requests, r.URL.Query())
-
-		switch len(requests) {
-		case 1:
-			// newest chunk — returns 2 traces
+		q := r.URL.Query()
+		rec.add(q)
+		// 90-min range → 6 chunks of 15 min each (range/6, no body parse).
+		switch q.Get("start") + "-" + q.Get("end") {
+		case "4500-5400":
 			w.Write([]byte(traceAPIResponse(2)))
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)